File: src/Entity/Continents.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Continents
 *
 * @ORM\Table(name="continents", uniqueConstraints={@ORM\UniqueConstraint(name="con_title", columns={"con_code"}), @ORM\UniqueConstraint(name="con_code", columns={"con_code"})})
 * @ORM\Entity
 */
class Continents
{
    /**
     * @var bool
     *
     * @ORM\Column(name="con_id", type="boolean", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $conId;

    /**
     * @var string
     *
     * @ORM\Column(name="con_code", type="string", length=3, nullable=false)
     */
    private $conCode;

    /**
     * @var string
     *
     * @ORM\Column(name="con_title", type="string", length=35, nullable=false)
     */
    private $conTitle;

    public function getConId(): ?bool
    {
        return $this->conId;
    }

    public function getConCode(): ?string
    {
        return $this->conCode;
    }

    public function setConCode(string $conCode): self
    {
        $this->conCode = $conCode;

        return $this;
    }

    public function getConTitle(): ?string
    {
        return $this->conTitle;
    }

    public function setConTitle(string $conTitle): self
    {
        $this->conTitle = $conTitle;

        return $this;
    }


}

File: src/main/java/models/Continents.java

package models;

import java.io.Serializable;
import java.util.Collection;
import javax.persistence.Basic;
import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.Id;
import javax.persistence.NamedQueries;
import javax.persistence.NamedQuery;
import javax.persistence.OneToMany;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;
import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlTransient;


@Entity
@Table(name = "continents")
@XmlRootElement
@NamedQueries({
    @NamedQuery(name = "Continents.findAll", query = "SELECT c FROM Continents c")})
public class Continents implements Serializable {

    private static final long serialVersionUID = 1L;
    @Id
    @Basic(optional = false)
    @NotNull
    @Column(name = "con_id")
    private Short conId;
    @Basic(optional = false)
    @NotNull
    @Size(min = 1, max = 3)
    @Column(name = "con_code")
    private String conCode;
    @Basic(optional = false)
    @NotNull
    @Size(min = 1, max = 35)
    @Column(name = "con_title")
    private String conTitle;
    @OneToMany(cascade = CascadeType.ALL, mappedBy = "ctrContinentId", fetch = FetchType.LAZY)
    private Collection<Countries> countriesCollection;

    public Continents() {
    }

    public Continents(Short conId) {
        this.conId = conId;
    }

    public Continents(Short conId, String conCode, String conTitle) {
        this.conId = conId;
        this.conCode = conCode;
        this.conTitle = conTitle;
    }

    public Short getConId() {
        return conId;
    }

    public void setConId(Short conId) {
        this.conId = conId;
    }

    public String getConCode() {
        return conCode;
    }

    public void setConCode(String conCode) {
        this.conCode = conCode;
    }

    public String getConTitle() {
        return conTitle;
    }

    public void setConTitle(String conTitle) {
        this.conTitle = conTitle;
    }

    @XmlTransient
    public Collection<Countries> getCountriesCollection() {
        return countriesCollection;
    }

    public void setCountriesCollection(Collection<Countries> countriesCollection) {
        this.countriesCollection = countriesCollection;
    }

    @Override
    public int hashCode() {
        int hash = 0;
        hash += (conId != null ? conId.hashCode() : 0);
        return hash;
    }

    @Override
    public boolean equals(Object object) {
        // TODO: Warning - this method won't work in the case the id fields are not set
        if (!(object instanceof Continents)) {
            return false;
        }
        Continents other = (Continents) object;
        if ((this.conId == null && other.conId != null) || (this.conId != null && !this.conId.equals(other.conId))) {
            return false;
        }
        return true;
    }

    @Override
    public String toString() {
        return "models.Continents[ conId=" + conId + " ]";
    }
    
}
